### PR TITLE
Use vst's ID instead of file path.

### PR DESCRIPTION
**Pull request by KottV**
Hi,

this is a hack and proof of concept for using plugin's ids.
vstplugin titlebar dropdown is working
loading old presets is working
spawning vsts on-the-fly by typing is not working

I still  think there is a room for improvements, at now it uses uniqueid only (as a list array index), but preferable way would be use a whole `PluginDespcription`

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,3 @@
-[submodule "libs/JUCE"]
-	path = libs/JUCE
-	url = https://github.com/juce-framework/JUCE
```
Comment (NoxyNixie) [inline .gitmodules]: I don't think this is intentional?

Comment (KottV) [reply .gitmodules]: yep

File: resource/help_rus.txt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        чтобы быть уверенным, что вы не упустите ни одного крутого момента.
+   * кнопка "save state": сохранить текущую компановку рабочего пространства в файл. комбинация клавиш: ctrl-s
+   * кнопка "load state": загрузить ранее сохраненные рабочие пространства. комбинация клавиш: ctrl-l
+
+горячие кнопки
+   * ctrl/command-'s': сохранить
+   * ctrl/command-'l': загрузить
+   * shift-'p': пауза обработки звука. это остановит весь аудиовыход. снова нажмите shift-'p', чтобы продолжить
+   * F2: переключить отображение всех элементов управления adsr в виде слайдеров 
+        (при перетаскавании патча от модуля модуляции в зону adsr переключается автоматически)
```
Comment (NoxyNixie) [inline resource/help_rus.txt:62]: Shouldn't this be part of a different PR?

File: resource/tooltips_rus.txt [hunk 3]
```diff
@@ -0,0 +1,1794 @@
+
```
Comment (NoxyNixie) [inline resource/tooltips_rus.txt:1]: Shouldn't this be part of a different PR?

File: resource/userdata_original/controllers/FCB1010_1.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (NoxyNixie) [inline resource/userdata_original/controllers/FCB1010_1.json:1]: Shouldn't this be part of a different PR?

File: resource/userdata_original/controllers/FCB1010_4.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
+{
```
Comment (NoxyNixie) [inline resource/userdata_original/controllers/FCB1010_4.json:1]: Shouldn't this be part of a different PR?

File: resource/userdata_original/controllers/FCB1010_9.json [hunk 6]
```diff
@@ -0,0 +1,124 @@
+{
```
Comment (NoxyNixie) [inline resource/userdata_original/controllers/FCB1010_9.json:1]: Shouldn't this be part of a different PR?

File: resource/userdata_original/controllers/SL MkII MIDI 1.json [hunk 7]
```diff
@@ -0,0 +1,109 @@
+{
```
Comment (NoxyNixie) [inline resource/userdata_original/controllers/SL MkII MIDI 1.json:1]: Shouldn't this be part of a different PR?

File: resource/help_rus.txt (hunk 2 above)
Comment (KottV) [reply resource/help_rus.txt:62]: it should, perhaps I forget that I added it in my master branch

**Review by NoxyNixie — COMMENTED** [review]
Not sure if this was an accidental thing or not but it seems you've added and removed more than your PR describes ;)

**Comment by KottV** [comment]
I'll rework these changes in a separate PRs

**Pull request closed without merging**